lib/php/lib/ClassLoader/ThriftClassLoader.php

```diff
@@ -101,8 +102,9 @@ public function register($prepend = false)
      */
     public function loadClass($class)
     {
-        if ((true === $this->apcu && ($file = $this->findFileInApcu($class))) or
-            ($file = $this->findFile($class))
+        if (
```

Comment:
PSR-12 suggest to write "if" in this style.
It future i will try to migrate php code to psr-12 and add code style check, but first of all we should write tests and migrate to new php version